.github/workflows/test_typing.yml

```diff
@@ -19,4 +19,5 @@ jobs:
           pip install -e .[all]
 
       - name: Run mypy on typed modules
+        if: ${{ false }}
```

Comment:
is this just skipping them?

Reply:
Yes. It just does not run this step. I can also just comment the lines...

Reply:
I guess I'm confused then if this PR is fixing the issues, or just not running type checks anymore

Reply:
Ah sorry I probably merged this and #4950

Reply:
```suggestion
```